app/controllers/idv/please_call_controller.rb

```diff
@@ -22,5 +28,18 @@ def confirm_fraud_pending
         redirect_to account_url
       end
     end
+
+    def in_person_proofing_enabled?
+      IdentityConfig.store.in_person_proofing_enabled
```

Comment:
In theory, if SP has not opted in the user should not have made it to this point but should we also be checking if in_person_proofing_enabled is also true on the SP?

Reply:
This page is rendered asynchronously and always after the user has completed the identity verification flow. If they have not opted in but have completed IdV and been flagged for fraud they should absolutely see this page.